polylogue/local_sync.py

```
@@ -95,6 +96,17 @@ def _is_up_to_date(source: Path, target: Path) -> bool:
 CLAUDE_EXPORTS_DEFAULT = EXPORTS_ROOT / "claude"
 
 
+class _NoopProgress:
+    def __enter__(self):
```

Ruff (0.15.9)

[warning] Missing return type annotation for special method `__enter__`

(ANN204)